site/content/en/docs/handbook/addons/ingress-dns.md

```diff
@@ -109,6 +109,7 @@ echo "server=/test/$(minikube ip)" >/etc/NetworkManager/dnsmasq.d/minikube.conf
 
 Restart Network Manager
 ```
+sudo rm /etc/resolv.conf # will be needed, at least on ubuntu 22.04
```

Comment:
this seems to be too invasive and not probably gonna destroy a lot of peoples workflows. I suggest that you provide a command that backs up the current reseolv.conf and then if user wants to revert that they can restore it.